### PR TITLE
[Merged by Bors] - Rename old SmartStream to SmartModule

### PR DESCRIPTION
**Pull request by nacardin**
To resolve https://github.com/infinyon/fluvio/issues/1858

### CODE REVIEW

**Comment by nacardin** [comment]
I can publish the creates once approved

File: crates/fluvio-smartmodule/README.md [hunk 1]
```diff
@@ -26,18 +26,18 @@ edition = "2018"
 crate-type = ['cdylib']
 
 [dependencies]
-fluvio-smartstream = "0.2.0"
+fluvio-smartmodule = "0.2.0"
```
Comment (sehz) [inline crates/fluvio-smartmodule/README.md:29]: should be 0.1.0

**Review by sehz — APPROVED** [review]
Excellent work! Fairly comprehensive.   There is a minor typo in doc but can fix after merge.

**Comment by sehz** [comment]
remove smartmodule from published crates until they are published.

**Comment by sehz** [comment]
Great!

**Pull request closed without merging**